### PR TITLE
Added example for ComplexNorm

### PR DESCRIPTION
**Pull request by radioactive11**
Added example for [ComplexNorm](https://pytorch.org/audio/stable/transforms.html#complexnorm) as mentioned in this [issue](https://github.com/pytorch/audio/issues/1564)

### CODE REVIEW

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -780,6 +780,11 @@ class ComplexNorm(torch.nn.Module):
 
     Args:
         power (float, optional): Power of the norm. (Default: to ``1.0``)
+
+    Example
+        >>> complex_tensor = ... #  Tensor shape of (…, complex=2)
+        >>> transform = transforms.ComplexNorm(power=2)
+        >>> complex_tensor = transform(complex_tensor)
```
Comment (nateanl) [inline torchaudio/transforms.py:787]:
```suggestion
        >>> complex_norm = transform(complex_tensor)
```

**Review by nateanl — CHANGES_REQUESTED** [review]
Hi @radioactive11! Sorry for my late reply. Just fix the nit and we can merge it to main branch.

File: torchaudio/transforms.py (hunk 1 above)
Comment (radioactive11) [reply torchaudio/transforms.py]: No problems @nateanl. I have made the changes as per your suggestion :)

**Review by nateanl — APPROVED** [review]
Thanks. LGTM!

**Pull request merged**